app/views/About.js

```diff
@@ -114,6 +114,7 @@ export const AboutScreen = ({ navigation }) => {
         <View style={styles.spacer} />
         <View style={styles.spacer} />
       </ScrollView>
+
```

Comment:
you can run `yarn format` locally to run the prettier formatter! which you will need to do to pass the validation checks.